automod2/automod2.py

```diff
@@ -318,6 +321,36 @@ def __init__(self, bot):
                                       ','.join(failed_whitelists), msg_content)
             await self.deleteAndReport(message, msg)
 
+    @automod2.command(pass_context=True, no_pm=True)
+    @checks.mod_or_permissions(manage_server=True)
+    async def autoemojis(self, ctx, key: str = None):
+        """Will automatically add a set of emojis to all messages sent in this channel.
+
+        Currently supported: tips
+
+        Include the text [noemojis] to suppress emoji addition"""

```

Comment:
The trailing """ should be on a newline.

Reply:
fixed